### PR TITLE
feat: add partners component

### PR DESCRIPTION
**Pull request by alexzavgorodnii**
Added component for partners card
image

image

### CODE REVIEW

**Comment by braindrain9** [comment]
Let's point this PR to `feat/init-structure` branch to see actual changes, once https://github.com/MecenatOrg/frontend/pull/3 is not merged yet.

**Comment by braindrain9** [comment]
please rename PR name to `feat: add partners component` to follow our style guide.

File: src/shared/components/Partner/styles.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const PartnerWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (braindrain9) [inline src/shared/components/Partner/styles.ts:6]: let's try to keep our styles as clean as possible and use bootstrap classes instead if possible,

e.g `d-flex align-items-center`

**Comment by braindrain9** [comment]
also, I think it's a good practice to add some screenshots of done work (partners in our case) into the PR description to give more context and in order do not build locally to see the result.

File: src/shared/components/Partner/styles.ts (hunk 1 above)
Comment (alexzavgorodnii) [reply src/shared/components/Partner/styles.ts]: Thank you for your review. 
Fixed at https://github.com/MecenatOrg/frontend/pull/7/commits/07f455e542f0c981a1b2a472dc1f9eb49e495e6c

**Comment by alexzavgorodnii** [comment]
> also, I think it's a good practice to add some screenshots of done work (partners in our case) into the PR description to give more context and in order do not build locally to see the result.

image

File: src/shared/components/Partner/styles.ts (hunk 1 above)
Comment (alexzavgorodnii) [reply src/shared/components/Partner/styles.ts:6]: I return old code because @lucypoly  say we mast don't use className

**Comment by braindrain9** [comment]
@alexzavgorodnii have you considered adding styles for mobile view?

Screenshot 2020-04-13 at 17 48 52

if yes, please attach screenshots. thanks

File: src/shared/components/Partner/component.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { PartnerWrapper, ImageElement } from './styles';
+
+type PartnerProps = {
+  imageSrc?: string;
+  imageAlt?: string;
+};
+
+const Partner: React.FC<PartnerProps> = (props: PartnerProps) => {
+  return (
+    <PartnerWrapper>
```
Comment (braindrain9) [inline src/shared/components/Partner/component.tsx:12]: Heading is missing

Comment (alexzavgorodnii) [reply src/shared/components/Partner/component.tsx:12]: What do you mean?

Comment (braindrain9) [reply src/shared/components/Partner/component.tsx:12]: Screenshot 2020-04-13 at 18 41 05

Comment (braindrain9) [reply src/shared/components/Partner/component.tsx:12]: 'Партнери' heading

Comment (alexzavgorodnii) [reply src/shared/components/Partner/component.tsx:12]: Oh, it's only the partner card component.

Comment (braindrain9) [reply src/shared/components/Partner/component.tsx:12]: okay, I think heading could be added as a separate shared component after UI kit implemented.

**Comment by alexzavgorodnii** [comment]
> @alexzavgorodnii have you considered adding styles for mobile view?
> 
> Screenshot 2020-04-13 at 17 48 52
> 
> if yes, please attach screenshots. thanks

Fixed.

File: src/shared/components/Partner/styles.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    box-shadow: 0px 20px 30px rgba(233, 229, 221, 0.6);
+  }
+`;
+export const ImageElement = styled.img`
+  display: block;
+  width: 140px;
+  height: auto;
+
+  @media (max-width: 576px) {
+    width: 88.45px;
```
Comment (braindrain9) [inline src/shared/components/Partner/styles.ts:20]: we should not set strict width here - make it flexible for any amount of items with `flexbox`.

Comment (olegdolenchuk) [reply src/shared/components/Partner/styles.ts:20]: I think you can use max-width: 88px; width: 100%;

Comment (alexzavgorodnii) [reply src/shared/components/Partner/styles.ts]: Fixed. I have taken padding from design. And made width for  100%.

image
image

File: src/shared/components/Partner/component.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+type PartnerProps = {
+  imageSrc?: string;
+  imageAlt?: string;
+};
+
+const Partner: React.FC<PartnerProps> = (props: PartnerProps) => {
+  return (
+    <PartnerWrapper className="d-flex align-items-center justify-content-center">
+      <ImageElement src={props.imageSrc} alt={props.imageAlt}></ImageElement>
```
[the remark this answers is not in the document]
Comment (alexzavgorodnii) [reply src/shared/components/Partner/component.tsx]: Sorry, I am already fixed it.

File: src/shared/components/Partner/component.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+import { PartnerWrapper, ImageElement } from './styles';
+
+type PartnerProps = {
+  imageSrc?: string;
+  imageAlt?: string;
+};
+
+const Partner: React.FC<PartnerProps> = (props: PartnerProps) => {
+  return (
+    <PartnerWrapper className="d-flex align-items-center justify-content-center">
```
[the remark this answers is not in the document]
Comment (alexzavgorodnii) [reply src/shared/components/Partner/component.tsx]: Already fixed

File: src/shared/components/Partner/component.tsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { PartnerWrapper, ImageElement } from './styles';
+
+type PartnerProps = {
+  imageSrc: string;
+  imageAlt: string;
+};
+
+const Partner: React.FC<PartnerProps> = (props: PartnerProps) => {
```
Comment (lucypoly) [inline src/shared/components/Partner/component.tsx:9]: I propose here `({imageSrc, imageAlt}: PartnerProps)`
And after `src={imageSrc} alt={imageAlt}`

Comment (alexzavgorodnii) [reply src/shared/components/Partner/component.tsx]: done

**Pull request merged**